sdk/src/metrics/state/temporal_metric_storage.cc

```diff
@@ -51,6 +51,36 @@ bool TemporalMetricStorage::buildMetrics(CollectorHandle *collector,
   AggregationTemporality aggregation_temporarily =
       collector->GetAggregationTemporality(instrument_descriptor_.type_);
 
+  // Fast path for single collector with delta temporality and counter, updown-counter, histogram
+  if (collectors.size() == 1 && collector->GetAggregationTemporality(
+                                    instrument_descriptor_.type_) == AggregationTemporality::kDelta)
```

Comment:
Nit: use the `aggregation_temporarily` local variable, from the previous line, in the if test.

Reply:
Thanks done.